com/uber/kafkaSpraynozzle/stats/StatsDReporter.java

```diff
@@ -0,0 +1,61 @@
+package com.uber.kafkaSpraynozzle.stats;
+
+import com.timgroup.statsd.NonBlockingStatsDClient;
+import com.timgroup.statsd.StatsDClient;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Statsd StatsReporter implementation
+ *
+ * To initialize this class you need to provide it the path to a .properties file containing:
```

Comment:
The only other thing I can think of is very Javascript-y.

```
-d localhost:8125,default.path.to.prepend
-d somestatsdserver:8125
```

First split the string by the comma to separate the prefix, if provided, the second value of the array being the prefix, then split the first value of the array by the colon to get the host and port pair.

This could live inside of the `try-catch` block so people have the option of doing either this properties file or inline in the command line (so repositories could use the former, one-off things on EC2 could use the latter).